src/sentry/migrations/0561_backfill_new_notification_tables.py

```diff
@@ -284,6 +287,8 @@ def backfill_notification_settings(apps, schema_editor):
                     },
                 )
 
+        related_settings.update(is_migrated=True)
```

Comment:
@snigdhas another gotcha with migrations, I don't think the `update` function is there because it's not in the base Django class that we get from `apps.get_model`. Do the classic .save instead.

Reply:
Also, probably better to update `setting` instead of `related_settings` if you think about it from a graph traversing problem.